### PR TITLE
Fix wrong parameter name in diffusion attention-mask error message

### PR DESCRIPTION
**Pull request by PratikDhanave**
## What

In `gemma/diffusion/_sampler.py`, `_make_global_attention_mask` validates its `num_valid_tokens` argument:

```python
if num_valid_tokens is None:
    raise ValueError(
        'num_valid_samples must be provided if cache_length is set.'
    )
```

The message refers to `num_valid_samples`, which is **not** a parameter of the function — the argument is named `num_valid_tokens`. Anyone hitting this error would look for a `num_valid_samples` argument that doesn't exist.

The sibling guard in `_make_block_local_attention_mask` already uses the correct name (`'num_valid_tokens must be provided ...'`).

## Change

Align the error message with the actual parameter name. Message-only; no behavior change.

### CODE REVIEW

**Comment by PratikDhanave** [comment]
Small fix: wrong parameter name in the diffusion attention-mask error message. CI green — ready for review.
